tests/net/ieee802154/fragment/src/main.c

```diff
@@ -287,6 +287,17 @@ static struct net_pkt *create_pkt(struct net_fragment_data *data)
 		}
 	}
 
+	/* Setup link layer addresses. */
+	static u16_t dummy_short_addr;
```

Comment:
no found of this, but up to you to change. Seems to be accepted in zephyr...

Reply:
> no found of this, but up to you to change

I didn't catch this up, could you elaborate?

Reply:
well it seems to be accepted to declare variable in a middle of a code block.

Reply:
Ah right, I can move it upwards, no prob